astor/code_gen.py

```diff
@@ -19,15 +19,18 @@
 
 import ast
 import sys
+import gc
 
 from .op_util import get_op_symbol, get_op_precedence, Precedence
 from .node_util import ExplicitNodeVisitor
-from .string_repr import pretty_string
-from .source_repr import pretty_source
+from .prettifier import StringLiteral, beautify
 
+###################################################################
+#               Main interface
+###################################################################
 
 def to_source(node, indent_with=' ' * 4, add_line_information=False,
-              pretty_string=pretty_string, pretty_source=pretty_source):
```

Comment:
This would potentially break the public API. We need a at least a short deprecation period (maybe remove them in 0.7?) to remove them from the `to_source` signature.